health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignDetails.js

```
@@ -0,0 +1,127 @@
+      </div>
+      <Footer
+        actionFields={[
+          <Button
+            icon="CheckCircleOutline"
+            label={t("HCM_CREATE_CAMPAIGN")}
+            onClick={() =>{}}
+            isDisabled={true}
+            type="button"
+            variation="primary"
+            className={"create-campaign-disable"}
+          />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)